packages/blade/src/components/NavigationBar/_decisions/decisions.md

```diff
@@ -0,0 +1,261 @@
+# NavigationBar - One Top Navigation Bar
+
+The navigation bar is positioned at the top of the screen that provides quick access to different products, search & user profile. 
+
+<img src="./top-nav-bar.png" width="100%" />
+
+## Links
+
+- [Figma - Top Navigation Bar](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=90311-235393&m=dev)
+- [Product Concept Note - Connected Navigation](https://docs.google.com/document/d/1MyCgIS8i3hfhYPiU94oNT0BwqFyNSa8fhVZLT1FlS3s/edit?usp=sharing) (Internal)
+
+## Anatomy & Components
+
+- NavigationBar
+  - NavigationBarBrand *(Logo or Brand Name)*
+  - NavigationBarContent *(Can contain any JSX)*
+    - TabNav *(Horizontal Navigation Bars)*
+      - TabNavItem
+  - NavigationBarActions *(Right Aligned Actions)*
+    - SearchInput
+    - Button
+    - Tooltip
+    - Menu
+      - Avatar
+      - MenuOverlay
+
+## Basic Usage
+
+```jsx
+// Desktop Navigation Bar
+<NavigationBar>
+  <NavigationBarBrand>
+    <img src="razorpay-logo" />
+  </NavigationBarBrand>
+  <NavigationBarContent>
+    <TabNav>
+      <TabNavItem as={RouterLink} isSelected href="/home" icon={HomeIcon} />
+      <TabNavItem as={RouterLink} href="/money">Money</TabNavItem>
+      <TabNavItem as={RouterLink} href="/payroll">Payroll</TabNavItem>
+      <Menu>
+        <TabNavItem trailingIcon={ChevronDown}>Payroll</TabNavItem>
+        <MenuOverlay>
+          { /* Other Menu Items */ }
```

Comment:
Do we not want control on how the items inside this menu should look like?

Reply:
Yeah, because that would require a new component to be built and exported from blade, rather it will be better if we let consumer handle this with Menu. Product requirement of that "others" menu can also change in future so best to keep it a bit flexible.